test_regress/t/t_flag_help.py

```diff
@@ -14,26 +14,22 @@
 # See also t_flag_version.py
 
 
-def check(interpreter, prog):
+def check(prog):
     logfile = test.obj_dir + "/t_help__" + os.path.basename(prog) + ".log"
 
-    test.run(fails=False,
-             cmd=[interpreter, prog, "--help"],
-             logfile=logfile,
-             tee=False,
-             verilator_run=True)
+    test.run(fails=False, cmd=[prog, "--help"], logfile=logfile, tee=False, verilator_run=True)
 
     test.file_grep(logfile, r'(DISTRIBUTION|usage:)')
 
 
-check("perl", os.environ["VERILATOR_ROOT"] + "/bin/verilator")
-check("perl", os.environ["VERILATOR_ROOT"] + "/bin/verilator_coverage")
+check(os.environ["VERILATOR_ROOT"] + "/bin/verilator")
+check(os.environ["VERILATOR_ROOT"] + "/bin/verilator_coverage")
 
-check("python3", os.environ["VERILATOR_ROOT"] + "/bin/verilator_ccache_report")
-check("python3", os.environ["VERILATOR_ROOT"] + "/bin/verilator_gantt")
-check("python3", os.environ["VERILATOR_ROOT"] + "/bin/verilator_profcfunc")
+check(os.environ["VERILATOR_ROOT"] + "/bin/verilator_ccache_report")
+check(os.environ["VERILATOR_ROOT"] + "/bin/verilator_gantt")
+check(os.environ["VERILATOR_ROOT"] + "/bin/verilator_profcfunc")
 
 if os.path.exists(os.environ["VERILATOR_ROOT"] + "/bin/verilator_difftree"):
-    check("python3", os.environ["VERILATOR_ROOT"] + "/bin/verilator_difftree")
+    check(os.environ["VERILATOR_ROOT"] + "/bin/verilator_difftree")
```

Comment:
This change is needed as `$VERIALTOR_ROOT/bin/verilator_gantt` and friends are now Perl redirect scripts in the installation, so python3 will not run them.

Not sure why we need to explicitly specify the interpreter here, it works just fine with executable scripts.. ?